code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -65,3 +68,21 @@
 		list(mode_name="stun", projectile_type=/obj/item/projectile/beam/stun, modifystate="nucgunstun", charge_cost = 240),
 		list(mode_name="lethal", projectile_type=/obj/item/projectile/beam, modifystate="nucgunkill", charge_cost = 480),
 		)
+
+//Functionally a Perun, but flavoured.
+/obj/item/weapon/gun/energy/gun/compact
+	name = "personal energy weapon"
+	desc = "The RayZar EW20 Cygnus personal energy weapon - or PEW - is Ward-Takahasi's entry into the variable capacity energy gun market. New users are advised to 'set RayZars to stun'."
+	description_fluff = "RayZar is Ward-Takahashi’s main consumer weapons brand, known for producing and licensing a wide variety of specialist energy weapons of various types and quality primarily for the civilian market."
+	icon_state = "PDWstun100"
+	item_state = null

```

Comment:
since it's already a subtype of `gun/energy/gun` the vars that aren't changed (`item_state`, `fire_delay`, `projectile_type' and `origin_tech`) are inherited and don't need to be redefined